main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
-
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
```

Comment:
Consider using a list of tuples to create `Genre` and `Actor` objects in a loop, as suggested in the checklist. This will make the code more concise and easier to maintain.